indico/MaKaC/webinterface/tpls/ConfDetailsFull.tpl

```diff
@@ -63,6 +63,8 @@
   </div>
   % endif
 
+  ${ template_hook('survey-announcement', event=conf) }
```

Comment:
Way too specific. What about `conference-home-announcements`? Not completely sure if "announcements" is a good fit though. @OmeGak  any suggestions?